metomi/rose/loc_handlers/rsync_remote_check.py

```diff
@@ -62,6 +62,9 @@ def main(path, str_blob, str_tree):
                 if filename.startswith("."):
                     continue
                 name = os.path.join(dirpath, filename)
+                # Bypass this check if rsyncing a broken symlink.
+                if os.path.islink(name) and not os.path.exists(name):
+                    continue
```

Comment:
Wouldn't it be better to use the same logic here as in checksum.py?

Reply:
Thus? (see changed code)